### ISSUE TITLE
New IPypublish version

### ISSUE THREAD

**Issue by chrisjsewell** [amichuda/Experimetrics-BITSS-Workshop#9]
Hey Aleksandr,

I didn't see an email address for you, so I'll leave this here :)
 
I noticed this workshop a while ago, and so I thought I'd let you know that I’ve now released v0.9 of ipypublish, which will be my last big release for a while.
When you get a chance I’d love to hear some feedback on it.
 
The main changes are detailed in https://ipypublish.readthedocs.io/en/latest/releases.html, but to summarise:
 
- Improved markdown cell conversion (see https://ipypublish.readthedocs.io/en/latest/markdown_cells.html)
- Integration of Jupytext, as a ‘pre-converter’ (for RMarkdown files)
- Addition of sphinx exporters and a sphinx extension
- Rewrite of the documentation, using the sphinx extension, and including Binder examples (see https://ipypublish.readthedocs.io/en/latest/code_cells.html)

 
Ta!
Chris

**Comment by lordflaron** [comment]
Hi Chris,

Thanks for letting me know about the email thing. I have added one to my profile.

I'm actually very familiar with IPypublish as I use it for converting my thesis notebooks into text when I need to present it. 

I was actually planning on introducing IPypublish at the workshop, but there wasn't enough time after making sure that Jupyter Notebooks were up and running on people's computer with the stata kernel (not by choice... although the stata_kernel is a really awesome implementation).

I think your work is exactly the thing that was missing from Jupyter. I recently finished converting my plugins to the new JSON format, although now for some reason, the PythonMarkdown preprocessor doesn't seem to be working. But I'll post an issue if I can't seem to figure it out.

Other than that though, I think it's a great tool, especially with Jupytext integration.

Thank you for reaching out and contributing this awesome package!
